.travis.yml

```diff
@@ -24,12 +35,9 @@ cache:
   directories:
     - $HOME/.composer/cache
 
-before_install:
-  - travis_retry composer self-update
-
 before_script:
-  - travis_retry composer install --no-interaction
+  - composer update --no-interaction --prefer-source $COMPOSER_FLAGS
 
 script:
-  - vendor/bin/phpunit --coverage-text
+  - if [[ $TEST_COVERAGE == 'true' ]]; then vendor/bin/phpunit --coverage-text ; else vendor/bin/phpunit ; fi
```

Comment:
What do you think about introducing `$PHPUNIT_FLAGS`? This way we can remove `if` from this statement.